crates/nu-command/src/formats/from/xlsx.rs

```diff
@@ -145,6 +146,12 @@ fn from_xlsx(
                             Data::Float(f) => Value::float(*f, head),
                             Data::Int(i) => Value::int(*i, head),
                             Data::Bool(b) => Value::bool(*b, head),
+                            Data::DateTime(d) => d
+                                .as_datetime()
+                                .and_then(|d| FixedOffset::east_opt(0).map(|o| (d, o)))
+                                .map(|(d, o)| DateTime::from_naive_utc_and_offset(d, o))
+                                .map(|d| Value::date(d, head))
```

Comment:
Excel's date time doesn't carry its own timezone info? (or as part of the metadata?)

Should this be Zulu time or local time?

Reply:
By default excel does not accound for timezones, its upto the sheet owner to manage this themselfs. What should be the default timezone, just utc?

Reply:
I'd prefer the local timezone

Reply:
For now I set it to local with a backup to Utc